src/CCapture.js

```diff
@@ -576,7 +576,7 @@ CCGIFEncoder.prototype.save = function( callback ) {
     this.callback = callback;
 
 	this.encoder.render();
-
+	return function() { this.encoder.abort() }.bind(this);

```

Comment:
Using a fat-arrow lambda style would be more compact:
`return () => this.encoder.abort();`

Reply:
that's true, though `function() {}` is consistent with the existing style in this file which does not use `=>`

Reply:
In case fat-arrow syntax isn't allowed, here's an alternative style w/o `.bind(this)`:
```js
  this.callback = callback;

  const enc = this.encoder;
  enc.render();

  return function() { enc.abort(); }
}
```